packages/components/src/DateTimePickers/views/HeaderTitle/HeaderTitle.component.js

```diff
@@ -21,6 +21,27 @@ function HeaderTitle(props) {
 	const pickerLocale = getPickerLocale(props.t);
 	const date = setYear(setMonth(new Date(0), props.monthIndex), props.year);
 	const label = format(date, 'MMMM YYYY', pickerLocale);
+	const yearLabel = format(date, 'YYYY', pickerLocale);
+	const monthLabel = format(date, 'MMMM', pickerLocale);
+
+	const withComponents = {
+		getComponent: key => {
+			if (key === 'YearPicker') {
+				return YearPicker;
+			}
+			throw new Error('Component not found');
+		},
+		components: {
+			itemsDropdown: [
+				{
+					id: 'year-picker-in-datetime-header',
+					component: 'YearPicker',
+					'data-feature': 'actiondropdown.items',
+					onSelect: props.onSelectYear,
+				},
+			],
+		},
+	};
```

Comment:
Oh nice use of the injection API but I guess with the API change happening in UI we should open children for that kind of business.
So I would add children in ActionDropdown.

Reply:
sorry, I don't understand.  🤣

Reply:
Added , thanks for your patch 😄